assemblies/pom.xml

```diff
@@ -129,8 +129,10 @@
               <phase>prepare-package</phase>
               <configuration>
                 <target>
-                  <!-- Make sure to deactivate JMX by default -->
+                  <!-- Make sure to deactivate JMX by default, comment out these three lines to enable JMX from the build -->
                   <replaceregexp file="target/assembly/bin/karaf" match="-Dcom.sun.management.jmxremote" replace="" byline="true"/>
+                  <replaceregexp file="target/assembly/bin/karaf.bat" match="-Dcom.sun.management.jmxremote" replace="" byline="true"/>
```

Comment:
This is specifically excluded in Opencast. So, this change has no effect at all.

```suggestion
```

Reply:
I removed that unneeded line.

Reply:
Oh, I could have committed the change right from your comment! That's cool. I'll do that next time.